### DISCUSSION TITLE
Pressure Advance < 0

### DISCUSSION THREAD

**Discussion by ditomaso** [OrcaSlicer/OrcaSlicer#145] [Q&A]
Tried the lines method of pressure advance and 0 was the best option but could still be a bit better. I'm unfamiliar with this setting -- is there a way to dial it back so I'm starting from a setting less than 0?

**Comment by SoftFever** [comment]
Negative PA is very unlikely.
I'd suggest you try the PA tower method, as the results of line method may be flawed if the initial layer quality is bad.
